examples/python/aws-eks/Pipfile

```diff
@@ -4,8 +4,8 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [requires]
-python_version = "3.7"
+python_version = "3.8"
 
 [packages]
 constructs = "~=3.0.0"
-cdktf = {path = "./../../../dist/python/cdktf-0.0.0-py3-none-any.whl"}
```

Comment:
What's the reason to remove the path to `dist` here? The examples should be built with the artifacts generated by `yarn build && yarn package`.

Reply:
Hi @skorfmann, thanks for pointing out - this change was unintended - Added a reversion commit